engine/testdata/fuzz/FuzzEnginePromQLSmithInstantQuery/010a0568eb7e1f6d

```diff
@@ -0,0 +1,7 @@
+go test fuzz v1
```

Comment:
Is this supposed to be commited?

Reply:
Yeah ( i mean arguably no but its a known fuzzing failure [ due to a defect i introduced to test fuzzing ] ); those files should eventually function as a regression suite that is built up through fuzzing, i think they are supposed to be committed.